recipes/libnice/all/conanfile.py

```diff
@@ -0,0 +1,109 @@
+import os
+from conan import ConanFile
+from conan.tools.meson import Meson, MesonToolchain
+from conan.tools.files import copy, get, rmdir, rename, chdir, replace_in_file
+from conan.tools.layout import basic_layout
+from conan.tools.gnu import PkgConfigDeps
+from conan.errors import ConanInvalidConfiguration
+
+
+class LibniceConan(ConanFile):
+    name = "libnice"
+    version = "0.1.19"
+    homepage = "https://libnice.freedesktop.org/"
+    license = "MPL-1.1 AND LGPL-2.1-only"
```

Comment:
```suggestion
    license = ("MPL-1.1", "LGPL-2.1-only")
```
Each license name should be a separated string to be parsed